File: Core/Models/Invoice.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Chinook.Core.Models
{
    public partial class Invoice
    {
        public Invoice()
        {
            InvoiceItems = new HashSet<InvoiceItem>();
        }

        public long InvoiceId { get; set; }
        public long CustomerId { get; set; }
        public byte[] InvoiceDate { get; set; }
        public string BillingAddress { get; set; }
        public string BillingCity { get; set; }
        public string BillingState { get; set; }
        public string BillingCountry { get; set; }
        public string BillingPostalCode { get; set; }
        public byte[] Total { get; set; }

        public virtual Customer Customer { get; set; }
        public virtual ICollection<InvoiceItem> InvoiceItems { get; set; }
    }
}

File: src/app/shared/models/invoice.ts
export class Invoice
{
	public invoiceId: number;
	public customerId: number;
	public invoiceDate: Date;
	public billingAddress: string;
	public billingCity: string;
	public billingState: string;
	public billingCountry: string;
	public billingPostalCode: string;
	public total: number;
}